core/src/com/mygdx/game/client/ecs/GameEngine.java

```diff
@@ -13,22 +15,21 @@
 public class GameEngine implements Updatable {
 
   private final Engine engine;
-  private final RenderSystem renderSystem;
-  private final UnitMovementSystem unitMovementSystem;
+  private final Array<EntitySystem> array = new Array<>();
```

Comment:
Może nazwij entitySystems albo gameSystems?

Reply:
Ta klasa miała mieć większą rolę tbh, na ten czas to w sumie racja. Wydaje mi się, że później jeszcze GameState logicznie byłoby tutaj trzymane i robiłoby "coś", ale na ten czas można zmienić tę nazwę.